.github/workflows/ghcr-retention.yml

```diff
@@ -0,0 +1,22 @@
+name: Delete untagged GHCR images
+on:
+  schedule:
+    - cron:  '17 10 * * TUE' # Tuesday morning at 10:17 UTC
+  workflow_dispatch:
```

Comment:
Necessary?

Reply:
I'm not sure which line was supposed to be highlighted, but assuming it's about the `workflow_dispatch` trigger, no it's not technically necessary but being able to manually run a scheduled event seemed reasonable.